tools/slang-reflection-test/slang-reflection-test-main.cpp

```
@@ -68,6 +68,9 @@ innerMain(Slang::StdWriters* stdWriters, SlangSession* session, int argc, const
     char const* appName = "slang-reflection-test";
     if (argc > 0)
         appName = argv[0];
+    // TODO: appName is set but never passed to performCompilationAndReflection.
```

Cppcheck (2.20.0)

[style] The function 'fread_s' is never used.

(unusedFunction)